src/pages/filter/ui/modal-edit/components/modal-filters/modal-filters.tsx

```diff
@@ -42,7 +41,12 @@ export const ModalFilters = observer((): ReactElement => {
         .includes(modalFiltersStore.searchValue.toLocaleLowerCase()),
   )
 
-  const chunks = createChunks(filteredGroupList, groupsPerPage)
+  const groupsPage = filteredGroupList.slice(
+    currentPage * groupsPerPage,
+    (currentPage + 1) * groupsPerPage,
+  )
+
+  const pagesNumbers = Math.ceil(filteredGroupList.length / groupsPerPage)
```

Comment:
you can move the whole bunch of logic into store getters

Reply:
Moved

Reply:
rename modal-filters.store.tsx to modal-filters.store.ts